webapp.yml

```diff
@@ -0,0 +1,71 @@
+# Description
+# ===========
+# This playbook create an Azure Linux Web app with Java container Tomcat 8.5,
```

Comment:
create -> creates
Tomcat 8.5 at the and a kind of doesn't fit

Reply:
fixed typo